controllers/remote/cluster_cache_tracker_test.go

```diff
@@ -98,9 +98,8 @@ func TestClusterCacheTracker(t *testing.T) {
 			t.Log("Creating a test cluster")
 			clusterA = &clusterv1.Cluster{
 				ObjectMeta: metav1.ObjectMeta{
-					Namespace:   ns.GetName(),
-					Name:        "test-cluster",
-					Annotations: make(map[string]string),
+					Namespace: ns.GetName(),
+					Name:      "test-cluster",
 				},
 			}
 			g.Expect(k8sClient.Create(ctx, clusterA)).To(Succeed())
```

Comment:
The `Create` call changes the empty map to nil.

I assume also a consequence of this change https://github.com/kubernetes-sigs/controller-runtime/pull/1640